### DISCUSSION TITLE
Duplicate events on different calendars not shown

### DISCUSSION THREAD

**Discussion by Unclearogre** [MMRIZE/MMM-CalendarExt3#155] [Q&A] [answered]
My wife and I each have a dental appointment on the same day at the same time.  I noticed that only one of them was being shown by CE3.  I changed the time of one to 15 minutes earlier and then both showed up.  I also found that when I copied an event from my calendar to my son's, only one showed up but when I changed the title of his event, both showed up.  

Is this expected behaviour?  

I would submit that it's not desirable.  In addition to the examples above, sometimes the "first day of school" or "last day of school" is the same for my wife and/or 3 kids.  If only one is shown, it's hard to know that the other kid would have the same last day.

**Comment by eouia** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/f0adb0b4-33ad-4000-ba3c-cce6c94daeeb)
It has been designated and introduced since 1.8.0(#117)
In contrast, someone may need remove duplicated events from multi-calendars. You can manage it with `skipDuplicated` options.

**Reply by Unclearogre** [reply]
Thanks!  I missed that somehow.  I appreciate the help!
